StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/IntentConfirmParams.swift

```diff
@@ -45,6 +45,51 @@ final class IntentConfirmParams {
         }
     }
 
+    var expandedPaymentSheetLabel: String {
+        switch paymentMethodType {
+        case .stripe(let stpPaymentMethodType):
+            switch stpPaymentMethodType {
+            case .card:
+                let brand = STPCardValidator.brand(for: paymentMethodParams.card)
+                return STPCardBrandUtilities.stringFrom(brand) ?? "Unknown"
+            case .USBankAccount:
+                // Use linked bank name if available, otherwise fallback to generic display name for bank
+                return financialConnectionsLinkedBank?.displayName ?? STPPaymentMethodType.USBankAccount.displayName
+            default:
+                // For all other payment method types just use the default label
+                return paymentSheetLabel
+            }
+        case .external:
+            return paymentSheetLabel
+        case .instantDebits:
+            return STPPaymentMethodType.link.displayName
+        case .linkCardBrand:
+            return STPPaymentMethodType.link.displayName
+        }
+    }
+
+    var paymentSheetSublabel: String? {
```

Comment:
Can we reuse the `STPPaymentMethod.paymentOptionLabel(confirmParams:)` extension method here? It already contains the handling for IBP, so no need for duplicate logic.

Reply:
Tried this out, but there are some structural differences between STPPaymentMethod and IntentConfirmParams that make this tricky.